### PR TITLE
Add package files and linter tests

### PR DESCRIPTION
**Pull request by jacobperron**
I will follow up by adding some actual code. Just wanted to get this boiler plate stuff out of the way.

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>rpyutils</name>
+  <version>0.1.0</version>
```
Comment (dirk-thomas) [inline package.xml:5]: Start with 0.0.0 until the first version is tagged.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+setup(
+    name=package_name,
+    version='0.1.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+    ],
+    zip_safe=False,
```
Comment (dirk-thomas) [inline setup.py:15]: Why is it not?

Comment (jacobperron) [reply setup.py]: It was copied from another setup.py, removed in e5ea4f1

File: package.xml (hunk 1 above)
Comment (jacobperron) [reply package.xml]: done a5a5e73

**Pull request merged**